packages/ra-ui-materialui/src/input/FileInput.stories.tsx

```diff
@@ -89,7 +89,7 @@ export const Multiple = () => (
 
 export const FullWidth = () => (
     <Wrapper>
-        <FileInput source="attachment" fullWidth>
+        <FileInput source="attachment" fullWidth={false}>
```

Comment:
**polish**: I do not see different width. IMO this story is useless

Reply:
Removed